Corzbank.Web/ClientApp/src/app/data/helpers/warning-notification/warning-notification.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-warning-notification',
+  templateUrl: './warning-notification.component.html',
+  styleUrls: ['./warning-notification.component.scss']
+})
+export class WarningNotificationComponent implements OnInit {
+
+
+  constructor(@Inject(MAT_DIALOG_DATA) public data: any) { }
+
+  textForNotification = this.data.name;
```

Comment:
variables initializing should be performed inside the ngOnInit

Reply:
resolved